covertest.py

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/env python
```

Comment:
Sort of - it's not just the test imports, it's also getting the `.Deps` of the each of the `.TestImports`, and it's also filtering out any of the values in `.Deps` that aren't in this particular subpackage or in the vendor package.